canonical-web-and-design/vanilla-framework
# Migrate docs to Flask

Migrates documentation site from static Jekyll site to flask web app

This is a feature branch, so code was already reviewed in relevant PRs.
This PR is meant for final QA before merging this to master.

### QA

- checkout feature branch
- `./run clean` to make sure to get rid of any artifacts from previous builds
- `./run`, wait for server to start
- go to localhost:8101 or [demo](https://vanilla-framework-canonical-web-and-design-pr-2782.run.demo.haus/)
- browse docs website
- check documentation pages
- check if codepen examples are rendering correctly
- check if navigation and side navigation work, if pages are marked selected properly
- go to examples listing page
- check different examples if they work as expected
- check standalone examples page
- check if standalone examples work as expected
